src/pretix/base/services/orders.py

```diff
@@ -1065,6 +1073,7 @@ def commit(self):
         if not self._operations:
             # Do nothing
             return
+        self._payment_fee_diff()
```

Comment:
This happens before the checks to incorporate a change in payment fees into `self._totaldiff`. I did not touch `self._recalculate_total_and_payment_fee()`, because it expects the change to be already applied to the order, since it is after `self._perform_operations()`.